lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -1027,6 +1026,16 @@ MStatus LayerManager::initialize()
         stat = addAttribute(identifier);
         CHECK_MSTATUS_AND_RETURN_IT(stat);
 
+        fileFormatId
```

Comment:
Since we don't share the same implementation of LayerManager, it would be good to add a test for this new addition to validate the core implementation as well. I won't make it a blocker for this change, but please open a new PR with it.